content/en/docs/topics/registries.md

```diff
@@ -11,17 +11,16 @@ stored and shared across OCI-based registries.
 
 ## Enabling OCI Support
 
-Currently OCI support is considered *experimental*.
+Prior to Helm v3.8.0, OCI support was considered *experimental* and needed to be
+enabled. As of v3.8.0 it is enable by default.
```

Comment:
> As of v3.8.0 it is enable by default.

As of v3.8.0 it is enable**d** by default.

Reply:
done